libevmasm/KnownState.cpp

```diff
@@ -100,6 +100,21 @@ KnownState::StoreOperation KnownState::feedItem(AssemblyItem const& _item, bool
 		feedItem(AssemblyItem(Instruction::POP), _copyItem);
 		return feedItem(AssemblyItem(Instruction::POP), _copyItem);
 	}
+	else if (_item.type() == VerbatimBytecode)
+	{
+		// Consume all arguments and place unknown return values on the stack.
+		m_stackElements.erase(
+			m_stackElements.upper_bound(m_stackHeight - static_cast<int>(_item.arguments())),
+			m_stackElements.end()
+		);
+		m_stackHeight -= static_cast<int>(_item.deposit());
+		for (size_t i = 0; i < _item.returnValues(); ++i)
+			m_stackElements[m_stackHeight - static_cast<int>(i)] = m_expressionClasses->newClass(_item.location());
```

Comment:
Can you explain what we are doing here?

Reply:
It is storing the return values as new symbolic variables on the stack.

Reply:
Let me rewrite it to use `setStackElement`